### ISSUE TITLE
Distribuição de tarefas

### ISSUE THREAD

**Issue by kvnol** [openschoolbr/forum#13]
Conversando no nosso grupo no Telegram com o @lflimeira e @nandomoreirame, chegamos numa ideia interessante para distribuição de tarefas que, a princípio, ficaria nos _**Projects**_ mas que concordamos que ficaria melhor uma lista **To Do** nos Readme.md de cada repositório.

**Comment by larismourullo** [comment]
Concordo com a lista no read.me. Pois vai chegar um momento que muitas pessoas podem ficar responsáveis por uma tarefa e isso pode se tornar um pouco confuso. E até mesmo pra gente caso precise saber quem ta sendo responsável por aquela tarefa. Seria uma forma de organização boa.

**Comment by nandomoreirame** [comment]
Minha sugestão não era nem colocar no Readme, mas sim na primeira _coluna_ (no trello chamamos de CardList) listar todas as tarefas para que todos possam visualizar o que falta pra finalizar, e as que estiverem em Progresso (in progress) altguém já está trabalhando. 

@kvnol se quiser eu posso ajudar a organizar isso, estou me especializando nessa área de organização de projetos.

**Comment by kvnol** [comment]
@nandomoreirame Então, no caso da sua sugestão ficaria no Projects aqui do GitHub, não é? O problema é que os projects nao ficam disponíveis para todos e a galera que está de fora não vai saber o que é mais importante para dar PR.

**Comment by nandomoreirame** [comment]
mas é padrão do Github deixar os Projects só para organizadores? Deve ter alguma config pra deixar publico.

A [WPBrasil](https://github.com/wpbrasil/) deixa os projects do [Odin publicos](https://github.com/wpbrasil/odin/projects).

Qualquer coisa podemos configurar as issues e usarmos elas com o [Waffle](https://waffle.io/), ele ajudaria muito nossa organização.

**Comment by nandomoreirame** [comment]
Assim da pra saber certinho em que pé que tá a tarefas e o andamento do projeto.

### Board do  projeto v2.4.0 do Odin Framework

![image](https://user-images.githubusercontent.com/1318271/29003292-61c1521a-7a8a-11e7-9dfa-1ced7da0d97b.png)

### Board do  projeto Waffle utilizando o Waffle

O próprio projeto da [Waffle tem seu board com as listas de atividades](https://waffle.io/waffleio/waffle.io)
![image](https://user-images.githubusercontent.com/1318271/29003320-d13d7344-7a8a-11e7-8a7c-c2f6e1178e36.png)

**Comment by kvnol** [comment]
Tu consegue ver isso, @nandomoreirame? Hoje eu não consigo. Se der certo, acho melhor irmos para os projects mesmo.

**Comment by nandomoreirame** [comment]
Sim, iniciei um modelo no repo [openschoolbr.github.io](https://github.com/openschoolbr/openschoolbr.github.io) chamado [v1.0.0 - Landing](https://github.com/openschoolbr/openschoolbr.github.io/projects/1)

A proposta seria:

- Para fazer (lista de todas as issues que é referente ao projeto ou curso)
- Em discussão (lista de todas as issues que ainda estão em discussão)
- Em progresso (lista de issues que foram puxadas por alguém que está trabalhando)
- Pull Requests (lista de issues que estão com algum Pull Request aberto)
- Finalizadas (lista de issues que foram finalizadas e que o PR já foi aceito na master)
- Fechadas (lista de issues que fechadas e que não foram feitas ou que tiveram PR recusados)

![image](https://user-images.githubusercontent.com/1318271/29003385-73a69952-7a8c-11e7-8df6-03e9ee97d2a2.png)

**Comment by khaosdoctor** [comment]
@nandomoreirame Você consegue tocar esta frente então? Ai colocamos no openschoolbr/sobre uma lista das pessoas que estão participando como foi descrito no #14 para todos saberem quem está participando e quem faz parte do que no projeto.

Ai fica muito mais fácil de dividir as coisas, ao invés de um cara fazer tudo, vamos tendo diversas pessoas que fazem coisas específicas e elas se reunem em uma _board_ para definir todas juntas cada frente :)

**Comment by nandomoreirame** [comment]
toco sim @khaosdoctor , mas cada projeto/curso precisa ser organizado pela pessoa responsável, como eu disse acima eu criei um modelo podemos discutir esse modelo para projeto e cursos.

**Comment by nandomoreirame** [comment]
O intetessante de usar o Waffle é que ele coloca labels nas issues em cada etapa e fecha se ela tiver em Done.

**Comment by khaosdoctor** [comment]
Beleza! Vamos colocar tudo no project? Vou deixar a triagem então para você adicionar porque pode ser que eu coloque coisa errada kkk

**Comment by nandomoreirame** [comment]
Então, a gente tem um problema de deixar no _Github Projetct_, temos que gerenciar a _Issue_ e o _Project_, porque o github nao faz isso automaticamente.

O que vocês acham de testarmos o **Waffle**, @kvnol e @khaosdoctor? 
Ele faz uma integração mais precisa com o Github, além de criar labels com o estado da atividade ele também fecha a issue quando é movida para Done. Isso pode nos ajudar a manter uma organização mais clara quanto ao andamento da atividade.

**Comment by khaosdoctor** [comment]
Eu realmente não conheço essa ferramenta, @nandomoreirame podemos iniciar um beta mais fechado só com alguns testers e depois abrir geral para todo mundo usar
